Symfony/CS/Tests/Fixer/Symfony/TrimArraySpacesFixerTest.php

```diff
@@ -0,0 +1,295 @@
+                "<?php \$foo = array( 'foo' );",
+            ),
+
+            array(
+                "<?php \$foo = ['foo'];",
+                "<?php \$foo = [ 'foo' ];",
+            ),
+
+            array(
+                "<?php \$foo = array();",
+                "<?php \$foo = array( );",
+            ),
+
+            array(
+                "<?php \$foo = [];",
+                "<?php \$foo = [ ];",
+            ),
+
+            array(
+                "<?php \$foo = array('foo', 'bar');",
+                "<?php \$foo = array( 'foo', 'bar' );",
+            ),
+
+            array(
+                "<?php \$foo = ['foo', 'bar'];",
+                "<?php \$foo = [ 'foo', 'bar' ];",
+            ),
+
+            array(
+                "<?php \$foo = array('foo', 'bar'); \$bar = array('foo', 'bar');",
+                "<?php \$foo = array( 'foo', 'bar' ); \$bar = array( 'foo', 'bar' );",
+            ),
+
+            array(
+                "<?php \$foo = ['foo', 'bar']; \$bar = ['foo', 'bar'];",
+                "<?php \$foo = [ 'foo', 'bar' ]; \$bar = [ 'foo', 'bar' ];",
+            ),
+
+            array(
+                "<?php \$foo = array('foo' => 'bar');",
+                "<?php \$foo = array( 'foo' => 'bar' );",
+            ),
+
+            array(
+                "<?php \$foo = ['foo' => 'bar'];",
+                "<?php \$foo = [ 'foo' => 'bar' ];",
+            ),
+
+            array(
+                "<?php \$foo = array('foo');",
+                "<?php \$foo = array( 'foo' );",
+            ),
+
+            array(
+                "<?php \$foo = ['foo'];",
+                "<?php \$foo = [ 'foo' ];",
+            ),
+
+            array(
+                "<?php \$foo = array(\$y ? true : false);",
+                "<?php \$foo = array( \$y ? true : false );",
+            ),
+
+            array(
+                "<?php \$foo = [\$y ? true : false];",
+                "<?php \$foo = [ \$y ? true : false ];",
+            ),
+
+            array(
+                "<?php \$foo = array(array('foo'), array('bar'));",
+                "<?php \$foo = array( array( 'foo' ), array( 'bar' ) );",
+            ),
+
+            array(
+                "<?php \$foo = [['foo'], ['bar']];",
+                "<?php \$foo = [ [ 'foo' ], [ 'bar' ] ];",
+            ),
+
+            array(
+                "<?php function(array \$foo = array('bar')) {}",
+                "<?php function(array \$foo = array( 'bar' )) {}",
+            ),
+
+            array(
+                "<?php function(array \$foo = ['bar']) {}",
+                "<?php function(array \$foo = [ 'bar' ]) {}",
+            ),
+
+            array(
+                "<?php \$foo = array(function() {return 'foo';});",
+                "<?php \$foo = array( function() {return 'foo';} );",
+            ),
+
+            array(
+                "<?php \$foo = [function() {return 'foo';}];",
+                "<?php \$foo = [ function() {return 'foo';} ];",
```

Comment:
can we include some more bad spacing to check only the correct stuff is changed? Same for the other test cases.